catalog/EDF/projects_EDF_MethaneSAT_public-preview_L3concentration.jsonnet

```diff
@@ -0,0 +1,297 @@
+// This is a generic template for an Earth Engine ee.ImageCollection dataset
```

Comment:
could you please remove the comments carried over from the template?

Reply:
done.  moved files to new location, will resolve this and we can continue over there.